src/tests/unit/tests/DetailsView/components/save-assessment-factory.test.tsx

```diff
@@ -1,14 +1,40 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { IMock, Mock } from 'typemoq';
+import { AssessmentDataFormatter } from 'common/assessment-data-formatter';
+import { FileURLProvider } from 'common/file-url-provider';
+import { AssessmentStoreData } from 'common/types/store-data/assessment-result-data';
 import {
     getSaveButtonForAssessment,
     getSaveButtonForFastPass,
+    SaveAssessmentFactoryDeps,
     SaveAssessmentFactoryProps,
 } from 'DetailsView/components/save-assessment-factory';
 
 describe('SaveAssessmentFactory', () => {
+    let deps: SaveAssessmentFactoryDeps;
     let props: SaveAssessmentFactoryProps;
 
+    beforeEach(() => {
+        const fileURLProviderMock: IMock<FileURLProvider> = Mock.ofType(FileURLProvider);
```

Comment:
If you'd like, you don't need to specify types here and with `assessmentDataFormatterMock` below since they can be inferred from the value you're setting:
```suggestion
        const fileURLProviderMock = Mock.ofType(FileURLProvider);
```

Not important, just a little more concise!